specs/player_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+  describe 'initialize' do
+    it 'takes player name' do
+      maja = Scrabble::Player.new("Maja")
+      maja.must_respond_to :name
+      maja.name.must_be_kind_of String
+
+      maja.must_respond_to :won
+      maja.won.must_equal false
+    end
+    it 'plays returns an array of words' do
+      player = Scrabble::Player.new("maja")
+
+      player.plays.must_be_instance_of Array
+      player.plays.length.must_equal 0
+    end
+  end
+
+  describe 'play method' do
+    it 'takes a word and add it to an array' do
+      brenda = Scrabble::Player.new("Brenda")
+
+      brenda.play("line")
+      brenda.play("fish")
+      brenda.play("knife")
+      brenda.play("cookie")
+
+      brenda.plays.include? "knife"
```

Comment:
You don't make any assertions here, which means as long as none of these methods raise an error, this test has no way to fail! You probably want to replace line 38 with something like

```ruby
brenda.plays.include?("knife").must_equal true

# or, minitest has a must_include expectation:

brenda.plays.must_include "knife"
```